src/Symfony/Component/Polyfill/Resources/stubs/Error.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class Error extends Exception implements Throwable
```

Comment:
this stub is misleading. ``Error`` in PHP 7 is *not* catched by a ``catch(\Exception $e)``, while it will catch your stub. this makes the stub unusable